cli/docker_start.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2086,SC2128,SC2048
```

Comment:
Was this really the best way to fix the shellcheck warnings?  The changes it recommends usually make your script safer, but I will trust that you made the right decision.

Reply:
The shellcheck concerns are highly conservative to cover all possible scenarios; in this case much of the script is dealing with output from docker, which is already well-formed and so many of these concerns didn't apply.  One warning was a case where it simply wanted a newline added to fit its preferred layout (not an error, simply a preference of the check author).  The final warning was concerned about a specific usage of an array variable which is a reasonable warning in general, but my use was specific and intentional and thus the warning was not necessary.